mapper-processor/pom.xml

```diff
@@ -94,6 +94,13 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.revapi</groupId>
+        <artifactId>revapi-maven-plugin</artifactId>
+        <configuration>
+          <skip>true</skip>
```

Comment:
The plugin fails because I forgot to add `revapi.json` files.

> I've added two skip rules

👍  for `mapper-processor` (it's purely internal). For `mapper-runtime` there are public types, such as the annotations, so we need to keep the plugin enabled and add a configuration file. I'll do this on `java2078`.